.travis.yml

```diff
@@ -12,7 +12,7 @@ jobs:
       script: pycodestyle --ignore=E402 backend
 
     - stage: Pylint
-      script: pylint backend
+      script: python scripts/kind_lint.py --targets backend --score 7
```

Comment:
change to:
python scripts/kind_lint.py --targets client backend --score 7

Reply:
+